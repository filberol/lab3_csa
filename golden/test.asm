code:
mov ax [null]
mov cx [max]
loop:
mov bx [three]
add ax bx
mov bx [two]
add ax bx
mov bx [one]
add ax bx
mov bx [three]
add ax bx
mov bx [one]
add ax bx
mov bx [two]
add ax bx
mov bx [three]
add ax bx
print bx
hlt
data:
three: 3
two: 2
one: 1
null: 0
max: 1000
