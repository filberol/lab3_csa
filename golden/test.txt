0x000a 10001001101000001000001000000000
0x000b 10001001101001001000001010000000
0x000c 10001001101000101000000010000000
0x000d 00000001100000000010000000000000
0x000e 10001001101000101000000100000000
0x000f 00000001100000000010000000000000
0x0010 10001001101000101000000110000000
0x0011 00000001100000000010000000000000
0x0012 10001001101000101000000010000000
0x0013 00000001100000000010000000000000
0x0014 10001001101000101000000110000000
0x0015 00000001100000000010000000000000
0x0016 10001001101000101000000100000000
0x0017 00000001100000000010000000000000
0x0018 10001001101000101000000010000000
0x0019 00000001100000000010000000000000
0x001a 11100110010000100000000000000000
0x001b 11110100001000000000000000000000
-
0x01 00000011
0x02 00000010
0x03 00000001
0x04 00000000
0x05 1111101000
